euth_wagtail/assets/scss/components/_dashboard-form-buttons.scss

```diff
@@ -0,0 +1,52 @@
+#dashboard-form-buttons {
```

Comment:
Styling with ids is for people who I consider CSS-Pros a no go. Although I can't give you specific issues that arise from it.